packages/pages/src/components/image/image.test.tsx

```diff
@@ -0,0 +1,219 @@
+import React from "react";
+import {Image, getImageUUID, handleLayout} from "./image";
+import {ImageLayout} from "./types";
+import { render, screen } from '@testing-library/react';
+
+const imgWidth = 20;
+const imgHeight = 10;
+const imgUUID = "uuid";
+const width = 200;
+const height = 100;
+const aspectRatio = 1;
+const image = {
+  image: {
+    width: imgWidth,
+    height: imgHeight,
+    url: `https://a.mktgcdn.com/p/${imgUUID}/2x1.jpg`,
+  },
+};
+
+/**
+ * @jest-environment node
```

Comment:
Same comment. Why did you set the env to jsdom globally and then just swap it back here? Also, I'm pretty sure this needs to be at the top of the file to work.

Reply:
Didn't get this to work and forgot to remove it lol